sdk/ml/azure-ai-ml/azure/ai/ml/_arm_deployments/arm_templates/workspace_base.json

```diff
@@ -437,32 +437,51 @@
                 "description": "Feature store online store config"
             }
         },
-        "offline_store_target": {
+        "offlineStoreStorageAccountOption": {
```

Comment:
do we have the same options for online store?

Reply:
We don't have it now because we don't create new online store for user, but I can add one just for consistency.

Reply:
I see. that is fine for now